src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -189,6 +172,7 @@ const UploadDocumentModal = ({ showModal, setShowModal }) => {
                 onClick={clearInputFields}
                 fullWidth
                 helpertext="Please enter your name"
```

Comment:
Not sure where this `helpertext` on the CANCEL button came from. I don't think it's needed.

Reply:
Good catch! Yeah, I don't think it's needed for this button. I'll remove it.